_posts/2017-03-27-open source update.md

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
We should probably change the filename to be published on a different date, as it won't be published on March 27, 2017.